docs/UserGuide.md

```diff
@@ -183,105 +188,116 @@ It appears when you execute the `help` command.
 ### Add Commands
 #### Adding a buyer to the database: `addbuyer`
 
-Adds a buyer to the database with relevant buyer information<br>
-Syntax: `addbuyer [-n NAME] [-ph PHONE] [-e EMAIL] [-a address] {-r PRICE RANGE} {-c CHARACTERISTICS} {-pr <HIGH, NORMAL, LOW>}`
+Adds a buyer to the database with relevant buyer information.<br>
+Syntax: `addbuyer -n NAME -ph PHONE -e EMAIL -a address [-r PRICE RANGE] [-c CHARACTERISTICS] [-pr PRIORITY<HIGH, NORMAL, LOW>]`
 
-The `-n` flag indicates the name of the buyer.<br>
+The `-n` flag indicates the buyer's name.<br>
 The `-ph` flag indicates the buyer’s phone number.<br>
 The `-e` flag indicates the buyer’s email.<br>
-The `-a` flag indicates the buyer’s house address.<br>
-The `-r` flag indicates the price range of properties that the buyer can accept.<br>
-The `-c` flag indicates the characteristics that the buyer is looking for in a property.<br>
+The `-a` flag indicates the buyer’s home address.<br>
+The `-r` flag indicates the price range of properties that the buyer can accept, denoted using `-`.<br>
```

Comment:
What do you mean by price range being denoted by `-`?

Reply:
using `-` to specify the range

Is this clearer?